hexrd/instrument/cylindrical_detector.py

```diff
@@ -117,7 +117,9 @@ def cart_to_dvecs(self,
                                                tVec_c=tvec_c,
                                                normalize=False)
 
-    def pixel_angles(self, origin=ct.zeros_3):
+    def pixel_angles(self, origin=ct.zeros_3, bvec: np.ndarray | None = None):
+        if bvec is None:
+            bvec = self.bvec
```

Comment:
We need to pass this `bvec` to the kwargs for `_pixel_angles` somehow. I'd suggest removing the bvec from the `self._pixel_angle_kwargs` and calling `_pixel_angles` like this:

```python
        return _pixel_angles(origin=origin, bvec=bvec, **self._pixel_angle_kwargs) 
```

Reply:
Done.